New Setup: Property Renovation

simonbbaumgartner opened the pull request:
Wüest Dimensions differentiates not just renovation: true/false. Wüest Dimensions has different refurbishment Component Types: total, interior_fittings_kitchen, interior_fittings_bathroom_sanitary, interior_fittings_floor_cover, interior_fittings_remaining building_envelope_pitched_roof, building_envelope_flat_roof, building_services_heat_production, building_services_heat_emission, buildings_services_electrical_ventilation_elevator, work_on_surroundings, supporting_structure, building_envelope_windows, building_envelope_facade_balcony

renovation should therefore be built as an own object with a list/array:

refurbishmentComponentType (string, enum: see above)
yearOfRenovation (similar to "renovationYear, but different name due to no duplicated fields)
renovationCost (see your Standard for "amount")

Adi-Moneypark commented:
In WP, it is possible to enter detailed or simple renovations. depending on which you choose, the amount of fields is different.

simonbbaumgartner commented:
To cover the simple renovation GUI of Wüest Partner, I added the following enum: interior_fittings / building_envelope / building_services 
To make the handling of the two different WP-renovation GUI clear, I adjusted the descpription.
